src/DotNetEnv/IValue.cs

```diff
@@ -46,12 +50,15 @@ public string GetValue ()
     public class ValueCalculator
     {
         public string Value { get; private set; }
+        public string RawValue { get; private set; }
 
-        public ValueCalculator (IEnumerable<IValue> values)
+        public ValueCalculator (IList<IValue> values)
```

Comment:
Why do you need this change?

Reply:
I have in mind that I somehow thought it was a good idea to "tell" the caller that we need to enumerate it inside here.
That removes the need within this method to enumerate it explicitly to be sure we have no multiple enumeration.

But as that adds the need for any caller to enumerate it before, while the caller anyway needs to expects the called function to enumerate it in any way, we can just stay with IEnumerable and do the enumeration inside of this method.

==> I will change that back